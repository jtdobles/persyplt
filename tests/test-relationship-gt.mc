void main(){
	int a = 3;
	if(a > 2){
		prints('a is greater than 2');
	}else{
		prints('a is not greater than 2');
	}
	if(a > 5){
		prints('a is greater than 5');
	}else{
		prints('a is not greater than 5');
	}
}