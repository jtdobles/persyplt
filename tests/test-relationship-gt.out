'a is greater than 2'
'a is not greater than 5'
